src/Scripting/Core/ScriptOptions.cs

```diff
@@ -92,6 +93,21 @@ private static ImmutableArray<MetadataReference> GetDefaultMetadataReferences()
         public ImmutableArray<string> Imports { get; private set; }
 
         /// <summary>
+        /// Specifies whether debugging symbols should be emitted.
+        /// </summary>
+        public bool EmitDebugInformation => DebugInformationFormat != null;
+
+        /// <summary>
+        /// <see cref="DebugInformationFormat"/> to be used for debugging symbols if needed
+        /// </summary>
+        public DebugInformationFormat? DebugInformationFormat { get; private set; } = null;
```

Comment:
>DebugInformationFormat? [](start = 15, length = 23)

Actually, I wonder if we need this option at all. We can detect whether the current platform supports Portable PDBs and only if it doesn't use Windows PDB.